### ISSUE TITLE
Obvious CI not found

### ISSUE THREAD

**Issue by scopatz** [conda-forge/conda-smithy#25]
Hello All, 

When testing a simple package (xz), I am seeing the following error in the circle CI build:

```console
./ci_support/run_docker_build.sh

Containers: 0
Images: 11
Storage Driver: btrfs
Execution Driver: lxc-1.0.7
Kernel Version: 3.14.28-031428-generic
Operating System: Ubuntu precise (12.04.5 LTS) (containerized)
CPUs: 2
Total Memory: 240.2 GiB
Name: box860
ID: 6D76:EKQV:CUD4:KSVV:UGDK:OJSW:AX7Y:CIIP:HRVF:VYAP:JM65:LONE
WARNING: No memory limit support
WARNING: No swap limit support
Unable to find image 'pelson/conda64_obvious_ci:latest' locally
Pulling repository pelson/conda64_obvious_ci

FATA[0000] Error: image pelson/conda64_obvious_ci:latest not found ./ci_support/run_docker_build.sh returned exit code 1
```

**Comment by scopatz** [comment]
Whoops, and here is the build: https://circleci.com/gh/scopatz/xz-feedstock/1

**Comment by ocefpaf** [comment]
@scopatz you need to update your `conda-smithy` (or you `run_docker_build.sh`).

The image location was changed in this commit: https://github.com/conda-forge/conda-smithy/commit/ab1b70c883dd48a1d569367b192370b826deb745

**Comment by scopatz** [comment]
@ocefpaf This was off of a clean clone and install of conda-forge/conda-smithy on a brand new machine.  That change seems to have been overridden on master

**Comment by ocefpaf** [comment]
Argh!  Let me fix that then.

**Comment by ocefpaf** [comment]
Wait... Unless I am missing something master seems to be pointing to the correct image:

https://github.com/conda-forge/conda-smithy/blob/master/conda_smithy/feedstock_content/circle.yml

**Comment by scopatz** [comment]
Yeah, I just noticed that too.  Maybe the latest_x86 is wrong?

**Comment by scopatz** [comment]
err latest_x64

**Comment by scopatz** [comment]
No the tag is right

**Comment by ocefpaf** [comment]
Nope.  The image on master (`pelson/obvious-ci:latest_x64`) is correct.  I just pulled it here locally.

Your configuration is trying to pull `pelson/conda64_obvious_ci:latest`, which does not exists.

I guess you somehow have some remnants of an old configuration.

**Comment by scopatz** [comment]
Ok - I think I have found it.  This line has yet-to-be update: https://github.com/conda-forge/conda-smithy/blob/master/conda_smithy/configure_feedstock.py#L145

**Comment by scopatz** [comment]
Yep #26 fixes this: https://circleci.com/gh/scopatz/xz-feedstock/4

**Comment by scopatz** [comment]
fixed by #26
